pkg/api/admin.go

```diff
@@ -33,6 +33,19 @@ func (hs *HTTPServer) AdminGetSettings(c *contextmodel.ReqContext) response.Resp
 	return response.JSON(http.StatusOK, settings)
 }
 
+func (hs *HTTPServer) AdminGetVerboseSettings(c *contextmodel.ReqContext) response.Response {
```

Comment:
if we want to expose this new endpoint in the swagger specification then we have to add the necessary [swagger annotations](https://github.com/grafana/grafana/blob/main/pkg/api/README.md#openapi-specifications).

Reply:
We decided to not expose this endpoint for now since the OSS implementation is missing. This is needed for an enterprise feature. 

https://github.com/grafana/grafana-enterprise/pull/4842